tmt/cli.py

```diff
@@ -579,19 +569,8 @@ def show(context, **kwargs):
 @name_filter_condition
 @verbose_debug_quiet
 def lint(context, **kwargs):
-    """
-    Check plans against the L2 metadata specification.
-
-    Regular expression can be used to filter plans by name.
-    Use '.' to select plans under the current working directory.
-    """
     tmt.Plan._save_context(context)
-    exit_code = 0
-    for plan in context.obj.tree.plans():
-        if not plan.lint():
-            exit_code = 1
-        echo()
-    raise SystemExit(exit_code)
+    lint_plan()
```

Comment:
It is not corresponding to lint for story and plan since this is the first try to implement `tmt lint`. As first I would like to have some feedback, so I can make the whole code better.

Reply:
The problem here is the removal of docstring which is used in the help message so now the help message is missing. I also think that it should be possible to use click to call the commands directly from `tmt lint` which would simplify the code. I will try it and send you an example if it works